libs/coin-framework/src/api/types.ts

```diff
@@ -6,15 +6,19 @@ export type BlockInfo = {
 
 export type Operation = {
   hash: string;
+  // TODO: check if useful
   address: string;
   type: string;
+  // TODO: check if we want Asset array
   value: bigint;
+  // TODO: check if we want Asset array
```

Comment:
there is not yet a usecase for an array. Is there one for tron ?

Reply:
removed